platform-api/src/main/java/io/datacater/core/deployment/K8Deployment.java

```diff
@@ -194,12 +194,13 @@ public void delete(UUID deploymentId) {
     String serviceName = StaticConfig.SERVICE_NAME_PREFIX + deploymentId;
     String configMapName = StaticConfig.CONFIGMAP_NAME_PREFIX + deploymentId;
     Boolean status =
-        client
+        !client
             .apps()
             .deployments()
             .inNamespace(StaticConfig.EnvironmentVariables.NAMESPACE)
             .withName(name)
-            .delete();
+            .delete()
+            .isEmpty();
```

Comment:
What is this change about? Is it related to the update?

Reply:
yes, the return type was changed with the upgrade to a list of the deleted items. The check now looks if the list is empty, then nothing was deleted. If it isn't empty, the deployment was deleted successfully